### ISSUE TITLE
Chris Week 8 - FRQ Showing 2

### ISSUE THREAD

**Issue by ChristopherDuroiu** [ChristopherDuroiu/usb-c#31]
Wiki with FRQ implementations, code, and takeaways:
https://github.com/ChristopherDuroiu/usb-c/wiki/Chris-FRQs

**Comment by ChristopherDuroiu** [comment]
| FRQ # | Rishi | Chris |
| --- | --- | --- |
| FRQ 4 | +++ Rishi's  code is very concise and his for loop and charAt shortcut make it very efficient --- His code is a little hard to follow without comments next to his variable. His naming is not very descriptive | My code is a little longer and less concise than Rishi's. That's because I constantly kept track of a text variable while Rishi only kept the index and went back at the very end to find the char |
| FRQ 6  | Rishi kept the String as is in order to find whether it ended in ing, and he also made better use of the necessary enhanced for loop | I reversed the string before operating on it, so I could send gni to the start. I had to real use for the enhanced for loop and just used it to make a char array which I converted back anyway |
| FRQ 9  | Rishi's code is extremely similar to mine and we have to same exact constructors, though he had a slightly different way of manipulating the parent method, which made no use of the override call | I used override for my second method and the rest is essentially the same |

Takeaways:
Frq 4: Rishi's code was a little more efficient than mine because he kept track of less things throughout the code, and used the thing he kept track of to find the rest at the very end
<br />
Frq 6: Reversing strings before manipulating them can make the index more easy to find conceptually. Enhanced for loop can be used to create an array holding characters
<br />
Frq 9: Basically the same code, both placing emphasis on the necessity to reinstantiate the variables from the parent class if they are declared private. 

FRQ Summary:

Some of my strengths are unique solutions for my code as shown through the "ing" identifier code as well as java shortcuts like charAt(). Some of my weaknesses are code efficiency and also naming of variables and methods, which sometimes lacks specificity.

**Comment by rpeddakama** [comment]
# Crossover Grading

**Total score: 9.8/10**
Unit 2: 1/1
Unit 3: 1/1
Unit 4: 1/1
Unit 5: 1/1
Unit 6: 0.8/1
Unit 7: 1/1
Unit 8: 1/1
Unit 9: 1/1
Unit 10: 1/1
At least 2 inputs: 1/1

Comments: All of Chris's FRQ's were properly implemented, they all worked on runtime and he had at least 2 with input functionality. That is why I gave 1/1 for all of this FRQ's because they had a good wiki, runtime, and source code. I removed 0.2 points for FRQ 6 because I felt that he could have made the code a little more efficient by getting rid of the for loops and using java's .substring() function instead.

Other grading:
The wiki with links to the code runtimes and takeaways was clear and showed good understanding of the college board topics. For the crossover grading, Chris did a good job of analyzing 3 frq's and seeing how they were similar/different and what he could do to improve his code.
